package.json

```diff
@@ -114,7 +114,7 @@
   "devDependencies": {
     "@types/debug": "4.1.7",
     "@types/jest": "26.0.24",
-    "@types/node": "16.11.43",
+    "@types/node": "^16.11.43",
```

Comment:
please don't change the semver direction of this package.

Reply:
Reverted